src/index/TextIndexBuilder.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2025, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Felix Meisen (fesemeisen@outlook.de)
+
+#ifndef QLEVER_SRC_INDEX_TEXTINDEXBUILDER_H
+#define QLEVER_SRC_INDEX_TEXTINDEXBUILDER_H
+
+#include "index/IndexImpl.h"
+
+class TextIndexBuilder : public IndexImpl {
```

Comment:
Add a short comment (this class contains all the code that is only required when building the fulltext index or something similar).